platform/os.go

```diff
@@ -0,0 +1,70 @@
+package platform
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// ReadFileByLines reads file line by line and return array of lines.
+func ReadFileByLines(filename string) ([]string, error) {
+	var (
+		lineStrArr []string
+	)
+
+	f, err := os.Open(filename)
+	if err != nil {
+		return nil, fmt.Errorf("Error opening %s file error %v", filename, err)
+	}
+
+	defer f.Close()
+
+	r := bufio.NewReader(f)
+
+	for {
+		lineStr, err := r.ReadString('\n')
+		if err != nil {
+			if err != io.EOF {
+				return nil, fmt.Errorf("Error reading %s file error %v", filename, err)
+			}
+
+			lineStrArr = append(lineStrArr, lineStr)
+			break
+		}
+
+		lineStrArr = append(lineStrArr, lineStr)
+	}
+
+	return lineStrArr, nil
+}
+
+func CheckIfFileExists(filepath string) (bool, error) {
+	_, err := os.Stat(filepath)
+	if err == nil {
+		return true, nil
+	}
+
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+
+	return true, err
```

Comment:
I know this is code moved from somewhere else, but should this be `false, err` instead? Although the client should be checking the error first, its common to return the zero value for all other returns other than the error.

Reply:
hmm may be what you saying is correct but i dont want to break anything by changing this.

Reply:
fair enough :)